docs/src/atomistic/outputs.rst

```diff
@@ -16,20 +16,105 @@ to this page.
 
 
 Energy
-------
-
-Energy (and its gradients: forces/virial) is associated with the ``"energy"``
-key in the model outputs, and must have the following metadata:
-
-- **keys**: the energy keys must have a single dimension named ``"_"``, with a
-  single entry set to ``0``. The energy is always a single block
-  :py:class:`metatensor.torch.TensorMap`.
-- **samples**: if doing ``per_atom`` output, the sample names must be
-  ``["system", "atom"]``, otherwise the sample names must be ``["system"]``.
-  ``"structure"`` must range from 0 to the number of systems given as input to
-  the model. ``"atom"`` must range between 0 and the number of atoms in the
-  corresponding system. If ``selected_atoms`` is provided, then only the
-  selected atoms for each system should be part of the samples.
-- **components**: the energy must not have any components.
-- **properties**: the energy must have a single property dimension named
-  ``"energy"``, with a single entry set to ``0``.
+^^^^^^
+
+Energy is associated with the ``"energy"`` key in the model outputs, and must
+have the following metadata:
+
+.. list-table:: Metadata for energy output
+  :widths: 2 3 7
+  :header-rows: 1
+
+  * - Metadata
+    - Names
```

Comment:
Names? Of who/what? Maybe name of the energy? Like U0?

Reply:
This is (one of) the syntax for tables in rst. Have you looked at how this is rendered? 

https://metatensor--514.org.readthedocs.build/en/514/atomistic/outputs.html